pyserini/util.py

```diff
@@ -89,17 +89,14 @@ def get_cache_home():
 def download_and_unpack_index(url, index_directory='indexes', force=False, verbose=True, prebuilt=False, md5=None):
     index_name = url.split('/')[-1]
     index_name = re.sub('''.tar.gz.*$''', '', index_name)
+    local_tarball = os.path.join(index_directory, f'{index_name}.tar.gz')
 
     if prebuilt:
         index_directory = os.path.join(get_cache_home(), 'indexes')
         index_path = os.path.join(index_directory, f'{index_name}.{md5}')
 
         if not os.path.exists(index_directory):
             os.makedirs(index_directory)
-
-        local_tarball = os.path.join(index_directory, f'{index_name}.tar.gz')
-        if os.path.exists(local_tarball):
```

Comment:
You may still need to keep this.
if the previous download has failed, then one must delete ~/.cache/pyserini/<index-md5> directory before trying to redownload the index. Otherwise the script throws error like
```
EOFError: Compressed file ended before the end-of-stream marker was reached
```

If the previous download is successful, the `local_tarball` should be deleted automatically. If there exists `local_tarball`, the previous download fails.

Reply:
Yea, good call!